tools/ci/production-upgrade

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# This test is for upgrading the production server
+set -e
+set -x
+
+# Uninstall supervisor and installing again before restarting server
+sudo apt-get remove supervisor
+sudo apt-get purge supervisor
+sudo apt-get install supervisor
+sudo service supervisor start
+
+# Perform the upgrade
+sudo -u zulip /home/zulip/deployments/current/scripts/restart-server
```

Comment:
Why do you restart the sever before doing the upgrade?  This seems like a surprising workflow.

Reply:
I was doing this as during the upgrade I got an error with few services unavailable or not running, so was trying to see if the restart script explicility performs a restart of certain processes. But still `process-fts-updates` does not start for some reason.

Reply:
Removed this!